core/silkworm/common/endian.hpp

```diff
@@ -72,26 +72,28 @@ as compiler intrinsics to swap bytes in 16-bit, 32-bit, and 64-bit integers resp
 namespace silkworm::endian {
 
 #if SILKWORM_BYTE_ORDER == SILKWORM_LITTLE_ENDIAN
-struct le {
-    static inline uint16_t uint16(uint16_t value) noexcept { return value; }
-    static inline uint32_t uint32(uint32_t value) noexcept { return value; }
-    static inline uint64_t uint64(uint64_t value) noexcept { return value; }
+struct LE {
+    static inline uint16_t load(uint16_t value) noexcept { return value; }
+    static inline uint32_t load(uint32_t value) noexcept { return value; }
+    static inline uint64_t load(uint64_t value) noexcept { return value; }
+    static inline intx::uint256 load(const intx::uint256& value) noexcept { return value; }
```

Comment:
I don't think the const& parameter of the last function is needed since it returns a value anyways and will be inlined. 
Then these 4 could be: 
```
template <class UnsignedInt>
static constexpr UnsignedInt load(UnsignedInt value) noexcept { return value; }
```